CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-### Added
+### Changed
+- Sort option in search-result is redesign to be dropdown button
+
+### Changed
```

Comment:
`Added`?

Reply:
`Added` is removed.

Reply:
what? but there's 2 `Changed` in the changelog, and before your commit, the changes bellow the line I commented were inside an `### Added` block